src/content/peerconnection/bandwidth/js/main.js

```diff
@@ -0,0 +1,207 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+/* global TimelineDataSeries, TimelineGraphView */
+
+'use strict';
+
+var remoteVideo = document.querySelector('video#remoteVideo');
+var localVideo = document.querySelector('video#localVideo');
+var callButton = document.querySelector('button#callButton');
+var hangupButton = document.querySelector('button#hangupButton');
+var bandwidthSelector = document.querySelector('select#bandwidth');
+hangupButton.disabled = true;
+callButton.onclick = call;
+hangupButton.onclick = hangup;
+
+var pc1;
+var pc2;
+var localStream;
+
+var bitrateGraph;
+var bitrateSeries;
+
+var packetGraph;
+var packetSeries;
+
+var lastResult;
+
+var offerOptions = {
+  offerToReceiveAudio: 0,
+  offerToReceiveVideo: 1
+};
+
+function gotStream(stream) {
+  hangupButton.disabled = false;
+  trace('Received local stream');
+  localStream = stream;
+  localVideo.srcObject = stream;
+  pc1.addStream(localStream);
+  trace('Adding Local Stream to peer connection');
+
+  pc1.createOffer(gotDescription1, onCreateSessionDescriptionError,
+      offerOptions);
+
+  bitrateSeries = new TimelineDataSeries();
+  bitrateGraph = new TimelineGraphView('bitrateGraph', 'bitrateCanvas');
+  bitrateGraph.updateEndDate();
+
+  packetSeries = new TimelineDataSeries();
+  packetGraph = new TimelineGraphView('packetGraph', 'packetCanvas');
+  packetGraph.updateEndDate();
+}
+
+function onCreateSessionDescriptionError(error) {
+  trace('Failed to create session description: ' + error.toString());
+}
+
+function call() {
+  callButton.disabled = true;
+  bandwidthSelector.disabled = false;
+  trace('Starting call');
+  var servers = null;
+  var pcConstraints = {
+    'optional': []
+  };
+  pc1 = new RTCPeerConnection(servers, pcConstraints);
+  trace('Created local peer connection object pc1');
+  pc1.onicecandidate = iceCallback1;
+  pc2 = new RTCPeerConnection(servers, pcConstraints);
+  trace('Created remote peer connection object pc2');
+  pc2.onicecandidate = iceCallback2;
+  pc2.onaddstream = gotRemoteStream;
+  trace('Requesting local stream');
+  navigator.mediaDevices.getUserMedia({
+    video: true
+  })
+  .then(gotStream)
+  .catch(function(e) {
+    alert('getUserMedia() error: ' + e.name);
+  });
+}
+
+function gotDescription1(desc) {
+  trace('Offer from pc1 \n' + desc.sdp);
+  pc1.setLocalDescription(desc, function() {
+    pc2.setRemoteDescription(desc, function() {
+      pc2.createAnswer(gotDescription2, onCreateSessionDescriptionError);
+    }, onSetSessionDescriptionError);
+  }, onSetSessionDescriptionError);
+}
+
+function gotDescription2(desc) {
+  pc2.setLocalDescription(desc, function() {
+    trace('Answer from pc2 \n' + desc.sdp);
+    // insert b=AS after c= line.
+    desc.sdp = desc.sdp.replace(/c=IN IP4 (.*)\r\n/,
```

Comment:
doesn't look like. I think because of interop concerns. There was this bug @phoglund found a while back.